src/app/common/utils/system-notifications/system.notifications.types.ts

```diff
@@ -1,4 +1,5 @@
 export interface IShowNotificationConfig {
     message: string,
     dismissBtnText?: string;
+    type?: string;
```

Comment:
Considering you should allow only string that you support (have proper CSS classes) it can't be `string` type. Otherwise someone can provide string `koza` and it won't work.

There is already created enum for notification types called `NotificationType` which you can use/extend for that.